### PR TITLE
extend the files method in the manager to filter for a given files

### PR DESCRIPTION
**Pull request by oaattia**
Opened new pull request regarding #50, as i tried to fix ci error on previous branch without no luck .

complete the method and add the missing tests
fix the missing spaces
adding doc to the missing method
add to missing command filter by language

closing #50

### CODE REVIEW

**Comment by oaattia** [comment]
@themsaid would you please help me why styleci is failing ? thanks

**Comment by themsaid** [comment]
Forget about styleCI, but travis is failing :) Means there are failing tests.

**Comment by oaattia** [comment]
Done, i didn't know that `array_where`'s parameters changed in laravel 5.3, i used `array_filter` instead

File: src/Commands/MissingCommand.php [hunk 1]
```diff
@@ -58,7 +58,11 @@ public function handle()
     {
         $this->info('Looking for missing translations...');
 
-        $languages = $this->manager->languages();
+	    if ( ! $this->option('lang')) {
```
Comment (mustafaaloko) [inline src/Commands/MissingCommand.php]: Remove the space before `i`. This is one of StyleCI failures.

File: src/Commands/MissingCommand.php [hunk 2]
```diff
@@ -169,6 +174,15 @@ private function getMissing(array $languages)
 
         $missing = array_merge($missing, $this->manager->getKeysExistingInALanguageButNotTheOther($values));
 
+	    if ( ! empty($languages)) {
```
Comment (mustafaaloko) [inline src/Commands/MissingCommand.php]: Remove the space before `i`.

Comment (oaattia) [inline src/Commands/MissingCommand.php]: @mustafaaloko 
Thanks for your help, removed the spaces as you mentioned, do you have any idea why CI failing in the tests ?

**Pull request closed without merging**